service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -32,15 +52,23 @@ default void bindServices(ServiceBinding<?>... serviceBindings) {
             }
         }
 
+        ServiceBinding<?> primaryServiceBinding = serviceBindings[0];
         // Bind the service info for the first one passed in
-        binder.bind(ServiceInfo.class).toProvider(new ServiceInfoProvider(serviceBindings[0].serviceInterface()));
+        binder.bind(ServiceInfo.class).toProvider(
+                new ServiceInfoProvider(
+                        primaryServiceBinding.serviceInterface(),
+                        Arrays
+                                .stream(serviceBindings)
+                                .map(ServiceBinding::serviceInterface)
+                                .toArray(Class[]::new)
+                ));
```

Comment:
Previously, `ServiceInfo` didn't contain any information about specific sub-services, just the name of the parent service.

I think this gets to the broader question of how we expect to use the ACL metadata in `ServiceInfo`. Should it replace the entire service registration in development? Should it be used in `JavadslServiceDiscovery` for tool support or by third-party integrations with service registries? Those currently look at all registered services, including `MetricsService`.

I was going to compare how the Scala side handles this, but it looks like `MetricsService` isn't automatically registered at all in the Scala version! I wonder why we make this different between the two. @jroper?